document/js-api/index.bs

```diff
@@ -700,12 +710,13 @@ To create a resizable memory buffer we perform the same steps as above, but also
     1. Assert: |map|[|memaddr|] [=map/exists=].
     1. Let |memory| be |map|[|memaddr|].
     1. Let |buffer| be |memory|.\[[BufferObject]].
-    1. If [=IsFixedLengthArrayBuffer=](|buffer|) is true,
-        1. If |memory|.\[[BufferObject]] is an {{ArrayBuffer}},
+    1. If [$IsFixedLengthArrayBuffer$](|buffer|) is true,
+        1. If [$IsSharedArrayBuffer$](|buffer|) is false,
             1. Perform [=!=] [$DetachArrayBuffer$](|buffer|, "WebAssembly.Memory").
-        1. Let |buffer| be the result of [=create a fixed length memory buffer|creating a fixed length memory buffer=] from |memaddr|.
-        1. Set |memory|.\[[BufferObject]] to |buffer|.
+        1. Let |newBuffer| be the result of [=create a fixed length memory buffer|creating a fixed length memory buffer=] from |memaddr|.
+        1. Set |memory|.\[[BufferObject]] to |newBuffer|.
     1. Otherwise,
+        1. Issue: Is this safe for a SharedArrayBuffer?
```

Comment:
Ah, this is a good point. It might not be right to operationally get the "length of |block|" in the SAB case this way.

Reply:
I'm currently super jetlagged from Wasmcon but I'll try to think more clearly about the right way to do this SAB case.